example/kotlinlib/web/3-hello-kotlinjs/src/foo/Hello.kt

```diff
@@ -12,5 +12,5 @@ fun main() {
 }
 
 fun hello(): String {
-    return createHTML().h1 { +"Hello World" }.toString()
-}
\ No newline at end of file
+    return createHTML(prettyPrint = false).h1 { text("Hello World") }.toString()
```

Comment:
What's the difference between these two snippets? Is the new one more idiomatic than the old one?

Reply:
It is effectively the same: `+` == `text` call. I find `+` syntax a bit weird, but in my other PR I used it. Pretty print is disabled just to avoid adding line breaks after the tag to simplify string assertion.